hathor/merged_mining/coordinator.py

```diff
@@ -1234,12 +1281,18 @@ def update_merged_block(self) -> None:
             output_script = create_output_script_btc(decode_address(self.payback_address_bitcoin))
         else:
             output_script = None
-        self.next_merged_job = MergedJob(
+        merged_job = MergedJob(
             self.hathor_coord_job,
             self.bitcoin_coord_job,
             output_script,
             self.is_next_job_clean(),
         )
+        # validate built job
+        block_proposal = merged_job.build_sample_block_proposal()
+        data = await self.bitcoin_rpc.verify_block_proposal(block=block_proposal)
+        self.log.debug('Validation result', data=data)
+        # job is valid
```

Comment:
Why is job valid here? If that's the case, the validation result will always confirm it?

Reply:
Not sure what I thought when I added that comment. It'll be valid if `data is None`. I'll improve this.

Reply:
Fixed. What do you think now?